### ISSUE TITLE
should this be implemented as a Moodle activity plugin?

### ISSUE THREAD

**Issue by danmarsden** [samuelcalegari/moodle-mod_mirotranslate#7]
This plugin appears to just embed any url that is passed into the url field as a frame within a page. This is functionality that is already supported with the Add "resource" activity type and is also possible using other methods such as filters etc.

Can you please clarify what functionality is provided by this plugin that is specific to mirotranslate and why users would need to use this plugin instead of the existing functions in Moodle?

**Comment by samuelcalegari** [comment]
Yes of course, this is a lite version, but in a next future some properties will be added to the external ressource (aka mirotranslate)
such as theme, secrets keys, colos, format etc.
